livre/eq/settings/CameraSettings.cpp

```diff
@@ -25,76 +25,99 @@
 #include <livre/core/maths/maths.h>
 #include <co/co.h>
 
+#include <algorithm>
+
 namespace livre
 {
 
 CameraSettings::CameraSettings()
-    : modelRotation_( Matrix4f::IDENTITY )
-    , cameraPosition_( 0.f, 0.f, 1.f )
 {
+    Matrix4f initial = Matrix4f::IDENTITY;
+    std::copy( &initial.array[0], &initial.array[0] + 16, getMatrix( ));
 }
 
-void CameraSettings::serialize( co::DataOStream& os, const uint64_t dirtyBits )
+void CameraSettings::spinModel( const float x, const float y )
 {
-    co::Serializable::serialize( os, dirtyBits );
-    os << modelRotation_ << cameraPosition_;
-}
+    if( x == 0.f && y == 0.f )
+        return;
 
-void CameraSettings::deserialize( co::DataIStream& is, const uint64_t dirtyBits )
-{
-    co::Serializable::deserialize( is, dirtyBits );
-    is >> modelRotation_ >> cameraPosition_;
-}
+    float matrixValues[16];
+    std::copy( getMatrix(), getMatrix() + 16, matrixValues );
 
-void CameraSettings::spinModel( const float x, const float y, const float z )
-{
-    if( x == 0.f && y == 0.f && z == 0.f )
-        return;
+    Matrix4f modelview( &matrixValues[0], &matrixValues[0] + 16 );
```

Comment:
Ok... Maybe we need the converting Matrix ctor after all, although it is tricky. Or maybe an explicit free function.. Leave as is, we can discuss later.